lib/lamby/handler.rb

```diff
@@ -72,7 +72,7 @@ def rack
       @rack = begin
         type = rack_option
         klass = Lamby::Rack.lookup type, @event
-        klass ? klass.new(@event, @context) : false
+        (klass && klass.handle?(@event)) ? klass.new(@event, @context) : false
```

Comment:
This was needed to avoid the dynamic work done in #92 being always forced. So now an event handler will double check the class can handle the rack event. This is how we fall thru to EventBridge, Runners, Lambdakiq, etc.